### PR TITLE
Fix issues with DNSMap tool as per the PTGUI Final Report

### PR DESCRIPTION
**Pull request by v-xl**
Fixed the following:

1. Spelling errors in the tooltip text
2. When selecting an external wordlist, the DDT GUI uses the incorrect command -S instead of -w.
3. When selecting results output to a CSV file, the DDT GUI uses the incorrect command -s instead of -c.
4. When selecting an IP address to ignore, the DDT GUI uses the incorrect command -t instead of -i.

### CODE REVIEW

**Comment by v-xl** [comment]
Just adding another note for my second commit from yesterday:

I had found that the command line args for DNSMap weren't working properly, and were either being ignored or causing errors upon the command execution. 

For example, when specifying a wordlist, it would not be used and when specifying a results .csv file it would also be ignored.

After doing some testing and making some changes, I had been able to get the tool functioning as intended. 

As can be seen in the commit, I've had to split into 2 separate args for each of the following:
- wordlistPath
- csvResultFile
- ipsToIgnore

File: src/components/DNSMap/DNSMap.tsx [hunk 1]
```diff
@@ -110,15 +110,15 @@ const DNSMap = () => {
         const args = [`${values.domain}`, "-d", `${values.delay}`];
 
         if (values.wordlistPath) {
-            args.push(`-S ${values.wordlistPath}`);
+            args.push(`-w`, `${values.wordlistPath}`);
```
Comment (Sakada-lim) [inline src/components/DNSMap/DNSMap.tsx:113]: This code work fine but i don't see why you would need to wrap your `values.wordlistPath` in a `''` as in the form the value is already define as a string. Same goes with the other if statement.

File: src/components/DNSMap/DNSMap.tsx [hunk 2]
```diff
@@ -110,15 +110,15 @@ const DNSMap = () => {
...
         }
 
         if (values.csvResultsFile) {
-            args.push(`-s ${values.csvResultsFile}`);
+            args.push(`-c`, `${values.csvResultsFile}`);
```
Comment (Sakada-lim) [inline src/components/DNSMap/DNSMap.tsx:117]: ### Suggestion for improvment
 For now the if statement is completely fine but if you want to improve the scalability of the code for future work you could look into expression such as ternary operator or a a better way of doing if statement.

**Review by Sakada-lim — COMMENTED** [review]
Overall the code look fine but can be improve if you have time:
### suggestion for improvement
improve the if statement for scalability

**Review by Sakada-lim — APPROVED** [review]

**Review by Phaellynn — APPROVED** [review]
Passed all functional and useability tests as per Gangas Marx
Screenshot 2023-11-26 210637

**Review by gangamarx — APPROVED** [review]
Passed all functional and useability tests. 
Test Execution Results - DNSMapping

**Pull request merged**